modules/uploads/scripts/get.ts

```diff
@@ -0,0 +1,52 @@
+import { ScriptContext } from "../_gen/scripts/get.ts";
+import { PrismaUploadWithOptionalFiles, prismaToOutput } from "../utils/types.ts";
+import { Upload } from "../utils/types.ts";
+
+export interface Request {
+	uploadIds: string[];
+	filesIncluded?: boolean;
```

Comment:
why not `includeFiles`?

Reply:
Mostly because I misread your original comment!